app/Models/user.ts

```diff
@@ -0,0 +1,34 @@
+import { DateTime } from 'luxon'
+import Hash from '@ioc:Adonis/Core/Hash'
+import {
+  column,
+  beforeSave,
+  BaseModel,
+} from '@ioc:Adonis/Lucid/Orm'
+
+export default class user extends BaseModel {
```

Comment:
```suggestion
export default class User extends BaseModel {
```
class name should be in PascalCase